pkgs/desktops/cinnamon/nemo/default.nix

```diff
@@ -0,0 +1,63 @@
+{ fetchFromGitHub
+, glib
+, gobject-introspection
+, meson
+, ninja
+, pkgconfig
+, stdenv
+, wrapGAppsHook
+, libxml2
+, cmake
+, gtk3
+, libnotify
+, cinnamon-desktop
+, xapps
+, libexif
+, exempi
+, intltool
+}:
+
+stdenv.mkDerivation rec {
+  pname = "nemo";
+  version = "4.4.1";
+  enableParallelBuilding = true;
+
+  src = fetchFromGitHub {
+    owner = "linuxmint";
+    repo = pname;
+    rev = version;
+    sha256 = "0sskq0rssxvna937md446x1489hkhxys1zq03hvl8asjqa259w2q";
+  };
+
+  patches = [ ./0001-patch-datadir.patch ./0002-patch-remove-update-mime-db-script.patch ];
+
+  NIX_CFLAGS_COMPILE = [ "-I${glib.dev}/include/gio-unix-2.0" ]; # TODO: https://github.com/NixOS/nixpkgs/issues/36468
+
+  buildInputs = [
+    glib
+    gtk3
+    libnotify
+    cinnamon-desktop
```

Comment:
`cinnamon-desktop` is currently not packaged, so this can't evaluate.

Did you mean to pull this out of #74242 first somehow?

Reply:
Oh yes.

Reply:
Oh yes :sweat_smile:

Reply:
Now it is ^^